app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, source_name, destination_name = "", "", ""
+    try:
+        cmd, source_name, destination_name = command.split()
+    except Exception:
+        print("Error in cmd!")
```

Comment:
Also, use `print()` is not very appropriate here; for example, if something goes wrong, a message will be printed, and the program will continue its execution, which will cause an error in the subsequent stages. Therefore, if the command is incorrect, I advise to immediately terminate the function's action.